docs/topics/mta-web-seeding-instances.adoc

```diff
@@ -0,0 +1,135 @@
+// Module included in the following assemblies:
+//
+// * docs/rules-development-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-seeding-instances_{context}"]
+= Seeding instances
+
+If you are a project architect you can configure the instances key parameters in the Controls window, before migration. The parameters can be added and edited as needed.
+The following parameters define applications, individuals, teams, verticals or areas within an organization affected or participating in the migration:
+
+* Stakeholders
+* Stakeholder groups
+* Job functions
+* Business services
+* Tag types
+* Tags
+
+You can create and configure an instance in any order. However, the suggested order below is the most efficient for creating stakeholders and tags.
```

Comment:
@CarmiWisemon 
Why 'create'? The instance has already been created. I would suggest "You can configure an instance in any order."